.github/ISSUE_TEMPLATE.md

```diff
@@ -7,3 +7,5 @@ Go to the "Preview" tab and click on the relevant issue type to be redirected to
 - [Help requests](?template=help-request.md)
 
 For other tasks (e.g. refactoring, adding tests, fixing typo), there is no template to follow. Simply provide a clear issue title and description of the work to be done.
+
+If you are reporting a security vulnerability, due to the sensitive nature of such report, you are kindly requested to report by emailing the team at teammates@comp.nus.edu.sg instead.
```

Comment:
... sensitive nature of such **a** report, you are kindly requested to refrain from opening a new issue. Instead, submit your report by emailing the team at teammates@comp.nus.edu.sg.